modules/caddyhttp/serverless/container.go

```
@@ -0,0 +1,298 @@
+	containerID := strings.TrimSpace(string(output))
+	if containerID == "" {
+		return nil, fmt.Errorf("docker run returned empty container ID")
+	}
+
+	cm.logger.Debug("container started", zap.String("container_id", containerID))
+
+	// Get container IP and port
+	container, err := cm.getContainerInfo(ctx, containerID, config.Port)
+	if err != nil {
+		// Clean up the container if we can't get its info
+		cm.stopContainerByID(ctx, containerID)
```

golangci-lint (1.64.8)

Error return value of `cm.stopContainerByID` is not checked

(errcheck)